### PR TITLE
Fix Python lint errors and alerts

### PR DESCRIPTION
**Pull request by manju956**
1. Remove unused import, unused variables
2. Remove logging sensitive data like IAS certifcate
3. Use is operator in place of == for null checks

Signed-off-by: manju956 <manjunath.a.c@intel.com>

### CODE REVIEW

**Comment by dcmiddle** [comment]
Nice cleanup, looks like there's more of the same if you want to hit these instances too:
```
$ grep -rs config_json_str --include='*.py' * | grep -v build
examples/common/python/shared_kv/remote_lmdb/test_lmdb_app.py:        config_json_str = json.dumps(config, indent=4)
examples/apps/echo/client/echo_client.py:		config_json_str = json.dumps(config)
examples/apps/heart_disease_eval/client/heart_gui.py:		config_json_str = json.dumps(config, indent=4)
examples/apps/generic_client/generic_client.py:		config_json_str = json.dumps(config)
tests/test_encrypted_data_encryption.py:                config_json_str = json.dumps(config)
tests/test_ias_attestation.py:                config_json_str = json.dumps(config)
tests/Demo.py:        config_json_str = json.dumps(config, indent=4)
```

**Comment by manju956** [comment]
> Nice cleanup, looks like there's more of the same if you want to hit these instances too:
> 
> ```
> $ grep -rs config_json_str --include='*.py' * | grep -v build
> examples/common/python/shared_kv/remote_lmdb/test_lmdb_app.py:        config_json_str = json.dumps(config, indent=4)
> examples/apps/echo/client/echo_client.py:		config_json_str = json.dumps(config)
> examples/apps/heart_disease_eval/client/heart_gui.py:		config_json_str = json.dumps(config, indent=4)
> examples/apps/generic_client/generic_client.py:		config_json_str = json.dumps(config)
> tests/test_encrypted_data_encryption.py:                config_json_str = json.dumps(config)
> tests/test_ias_attestation.py:                config_json_str = json.dumps(config)
> tests/Demo.py:        config_json_str = json.dumps(config, indent=4)
> ```

Sure, will cleanup these as well.

File: examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def __validate_input_worker_status(self, params):
         try:
             worker_status = params["status"]
             status = int(worker_status)
-            wo_status = WorkerStatus(status)
+            WorkerStatus(status)
```
Comment (manojgop) [inline examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py]: Why do we need this -> WorkerStatus(status)

Comment (manju956) [reply examples/common/python/connectors/direct/tcs_listener/tcs_worker_registry_handler.py]: this is to check if status value is within the value supported by enum WorkerStatus.

**Pull request merged**